packages/core/package.json

```diff
@@ -61,7 +61,7 @@
     "@blueprintjs/test-commons": "^0.8.1",
     "enzyme": "^3.3.0",
     "karma": "^3.1.4",
-    "mocha": "^4.1.0",
+    "mocha": "^5.2.0",
```

Comment:
For the record, looks like the only break in Mocha 5.x was to drop support for IE9 and IE10, which seems fine. Here are the [release notes](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#500--2018-01-17).